<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Connection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/program.png</normaloff>../icons/program.png</iconset>
  </property>
  <layout class="QGridLayout" name="DialogLayout" rowstretch="0,0,0" columnstretch="1,3">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="IPAddressLabel">
     <property name="text">
      <string>IP Address:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="PortLabel">
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="IPAddressWidget" native="true">
     <widget class="QLineEdit" name="IPSegment1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>32</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="IPSegment2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>32</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="IPSegment3">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>32</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="IPSegment4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>32</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Dot1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>32</y>
        <width>5</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Dot2">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>32</y>
        <width>5</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Dot3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>32</y>
        <width>5</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="ButtonsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddConnectionButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add connection</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="PortWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLineEdit" name="PortSegment">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>51492</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IsDefaultPort">
        <property name="toolTip">
         <string>Sets the port to default value (51492)</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Default Port</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddConnectionButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IsDefaultPort</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>EnablePort(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PortSegment</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>CheckPort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>EnablePort(bool)</slot>
  <slot>CheckPort()</slot>
 </slots>
</ui>
